lab_1_keywords_tfidf/main.py

```diff
@@ -12,12 +14,21 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
     Parameters:
     text (str): Original text
 
-    Returns:
+    Returns:  
```

Comment:
also please discard changes from docstrings

Reply:
What do you mean?